src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/judgeapproval/ApproveDraftOrdersSubmittedHandlerTest.java

```diff
@@ -66,9 +67,10 @@ void shouldInvokeNotificationServiceForRefusalOrders(DraftOrdersWrapper draftOrd
             .thenReturn(expectedNotificationRequest);
 
         // Act
-        handler.handle(callbackRequest, AUTH_TOKEN);
+        var response = handler.handle(callbackRequest, AUTH_TOKEN);
 
         // Assert
+        assertThat(response.getConfirmationHeader()).isEqualTo("# Draft orders reviewed");
```

Comment:
Why don't you test the response confirmation body?

Reply:
I was reviewing this yesterday. The only thing we do in the **submittedHandler** is set the body with whatever is in `ApproveOrdersConfirmationBody`:

`        String confirmationBody = caseDetails.getData().getDraftOrdersWrapper().getApproveOrdersConfirmationBody();`

I can add an assertion in this file to verify that the confirmation body is correctly assigned.

To thoroughly test whether the confirmation body is generated correctly, we’d need a detailed test in ApproveOrderServiceTest, as that’s where the logic for building the confirmation body resides. I’ve already written some tests for this and have them ready to go. However, there’s an issue with asserting the body that I’ll explain once I’ve committed the changes.